### ISSUE TITLE
[EN][Quest][The Storm Peaks] The Last of Her Kind

### ISSUE THREAD

**Issue by Tiennshan** [Atlantiss/BugTracker#4309]
http://www.wowhead.com/quest=12983/the-last-of-her-kind

There's no bear inside cave.
Phasing bug.

**Comment by Scrimish** [comment]
Bear is in side, but phasing with disguise in that quest only started to be dodgy. Before u reach a cave where the bear is u losing disguise buff which making npc hostile. Quest is important it's blocking hole chain connected to hodir's reputation.

**Comment by Helbringer** [comment]
Yes i tried this quest too and phasing makes lots of movs dissapear and your disguise too. Bear is no where to be seen.
![wowscrnshot_090215_172437](https://cloud.githubusercontent.com/assets/10423624/9637329/0248ddca-5198-11e5-9b91-81367d35d2eb.jpg)

**Comment by morgs71** [comment]
Possible fix from TrinityCoreDB

https://github.com/TrinityCore/TrinityCore/blob/3bc3b67a8fae059e1019446b09882dfa7fcd11a0/sql/old/3.3.5a/TDB42_to_TDB43_updates/world/2011_09_14_01_world_quest.sql

**Comment by artamedan** [comment]
The bug still persists! You lose Brunnhildar disguise near the cave. No Icemaw Matriarch bear inside the cave. Quest is very important for Sons of Hodir reputation chain.
Atlantiss should update the WotLK database, offered by Sunwell where all of the quests were working.

**Comment by Malanie** [comment]
any chance that this bug is solved in the near future?

**Comment by Destroman** [comment]
The Bear is inside the cave and u can ride him as well. The problem is that the bear should automatically walk back to the quest giver. I could control it by myself and run back to the quest giver but the quest didnt finish.

**Comment by Volcanicus** [comment]
Quick fix:
DO NOT CLICK THE BEAR.

Stand in front of her near the tip of her nose and you will auto-mount and auto-run back to hand in the quest.

Clicking the bear does not work and neither does running it back to the quest giver.
